client-spark/common/src/main/java/org/apache/spark/shuffle/celeborn/SortBasedPusher.java

```diff
@@ -61,6 +61,7 @@ public class SortBasedPusher extends MemoryConsumer {
   private final int pushBufferMaxSize;
   private final long pushSortMemoryThreshold;
   private final int shuffleId;
+  private final int sparkShuffleId;
```

Comment:
unused?

Reply:
no, this is a bug, it is fixed in latest commit